client/src/document/baseline-indicator.tsx

```diff
@@ -174,8 +174,9 @@ export function BaselineIndicator({ status }: { status: SupportStatus }) {
         )}
         <ul>
           <li>
+            {/* eslint-disable-next-line react/jsx-no-target-blank */}
```

Comment:
Why is this needed?

Reply:
eslint can't analyze dynamic links to determine if they're local or external, so fails safe and enables the rule for them: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md#enforcedynamiclinks

I reckon it would be useful to have referrer data on the glossary page to see which pages users are coming from

Reply:
Okay, all good.